### PR TITLE
Aviation Security (SO18) Borough

### PR DESCRIPTION
**Pull request by TomWFox**
- [x] Write up for SO18 & Hillingdon exploration
- [x] Add assets folder (can put images etc we want to use in the book)
- [x] Improve table of contents structure

### CODE REVIEW

**Comment by TomWFox** [comment]
@palpatel0 have a look here for inspiration on what a write up can look like. It's not perfect but it doesn't have to be... the important thing is that the reader can understand what decisions you have made, why and what do the results tell you... they should be able to follow along and not have loads of questions about what you're trying to show and what your code is doing.

File: _toc.yml [hunk 1]
```diff
@@ -3,7 +3,12 @@
 
 format: jb-book
 root: pages/intro
-chapters:
-- file: pages/markdown
-- file: pages/markdown-notebooks
-- file: pages/notebooks
+parts:
+  - caption: Explortation
```
Comment (palpatel0) [inline _toc.yml:7]:
```suggestion
  - caption: Exploration
```

Comment (TomWFox) [reply _toc.yml:7]: nice catch 👍🏻

**Review by palpatel0 — APPROVED** [review]
Looks good. Interesting thoughts :)

**Comment by TomWFox** [comment]
@mysteryvishal will merge in the morning in case you want to have a look as well...

**Pull request merged**